src/identity/components/IdentityLayout.tsx

```diff
@@ -55,6 +123,32 @@ export function IdentityLayout({ children, className }: IdentityLayoutReact) {
           </div>
         )}
       </div>
+      {showPopover && (
+        <div
+          className={cn(background.inverse, color.foreground, 'absolute z-10')}
+          style={{
```

Comment:
I figured this was done in a rush, which makes sense.
Let's make sure to have it as tailwind, as cleanup.

cc @kyhyco